tf-psa-crypto/tests/scripts/components-basic-checks.sh 

```diff
@@ -0,0 +1,15 @@
+# components-basic-checks.sh
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+# This file contains test components that are executed by all.sh
+
+################################################################
+#### Basic checks
+################################################################
+
+component_tf_psa_crypto_check_files () {
+    msg "Check: file sanity checks (permissions, encodings)" # < 1s
+    ../tests/scripts/check_files.py
```

Comment:
Repo split + check_files.py moved to the framework:  this PR needs now to be split into a framework PR and a TF-PSA-Crypto PR.